R/page.R

```diff
@@ -65,8 +74,81 @@ page_fixed <- function(..., title = NULL, theme = bs_theme(), lang = NULL) {
 #'   height on mobile devices (i.e., narrow windows).
 #' @param gap A [CSS length unit][htmltools::validateCssUnit()] defining the
 #'   `gap` (i.e., spacing) between elements provided to `...`.
+#' @inheritParams page
+#'
+#' @seealso [card()] for wrapping outputs in the 'main' content area.
+#' @seealso [value_box()] for highlighting values.
+#' @seealso [layout_columns()] and [layout_column_wrap()] for laying out content
+#'   into rows and columns.
+#'
+#' @examplesIf rlang::is_interactive()
+#'
+#' library(shiny)
+#' library(ggplot2)
+#'
+#' ui <- page_fillable(
+#'   h1("Example", code("mtcars"), "dashboard"),
+#'   layout_columns(
+#'     card(
+#'       full_screen = TRUE,
+#'       card_header("Number of forward gears"),
+#'       plotOutput("gear")
+#'     ),
+#'     card(
+#'       full_screen = TRUE,
+#'       card_header("Number of carburetors"),
+#'       plotOutput("carb")
+#'     )
+#'   ),
+#'   card(
+#'     full_screen = TRUE,
+#'     card_header("Weight vs. Quarter Mile Time"),
+#'     layout_sidebar(
+#'       sidebar = sidebar(
+#'         varSelectInput("var_x", "Compare to qsec:", mtcars[-7], "wt"),
+#'         varSelectInput("color", "Color by:", mtcars[-7], "cyl"),
+#'         position = "right"
+#'       ),
+#'       plotOutput("var_vs_qsec")
+#'     )
+#'   )
+#' )
+#'
+#' server <- function(input, output) {
+#'   for (var in c("cyl", "vs", "am", "gear", "carb")) {
+#'     mtcars[[var]] <- as.factor(mtcars[[var]])
+#'   }
+#'
+#'   output$gear <- renderPlot({
+#'     ggplot(mtcars, aes(gear)) + geom_bar()
+#'   })
+#'
+#'   output$carb <- renderPlot({
+#'     ggplot(mtcars, aes(carb)) + geom_bar()
+#'   })
+#'
+#'   output$var_vs_qsec <- renderPlot({
+#'     req(input$var_x, input$color)
+#'
+#'     ggplot(mtcars) +
+#'       aes(y = qsec, x = !!input$var_x, color = !!input$color) +
+#'       geom_point()
+#'   })
+#' }
+#'
+#' shinyApp(ui, server)
```

Comment:
Nice! Maybe once `shiny::runExample()` gets an extension point this would be a good candidate for an example?